pkg/kube/activity.go

```diff
@@ -195,6 +194,18 @@ func (k *PipelineActivityKey) GetOrCreate(activities typev1.PipelineActivityInte
 	}
 }
 
+func updateActivity(k *PipelineActivityKey, activity *v1.PipelineActivity) {
+	if activity.Labels == nil {
+		activity.Labels = make(map[string]string, 4)
+	}
+
+	updateActivitySpec(k, &activity.Spec)
+
+	activity.Labels[v1.LabelSourceRepository] = activity.RepositoryName()
```

Comment:
I think this will not work as intended if an app is imported (or created) using an `AppName` that differs from the git repo name